packages/bundle/src/adaptiveCards/Styles/adaptiveCardHostConfig.js

```diff
@@ -1,31 +1,32 @@
 import { defaultStyleOptions } from 'botframework-webchat-component';
 // https://docs.microsoft.com/en-us/adaptive-cards/rendering-cards/host-config
 
-export default ({
-  accent,
-  bubbleTextColor,
-  cardEmphasisBackgroundColor,
-  primaryFont,
-  subtle
-} = defaultStyleOptions) => ({
+export default ({ accent, primaryFont, subtle } = defaultStyleOptions) => ({
   containerStyles: {
     default: {
+      backgroundColor: null,
       foregroundColors: {
         default: {
-          default: bubbleTextColor,
-          subtle
+          default: null,
+          subtle: subtle
         },
         accent: {
-          default: accent,
+          default: null,
           subtle: accent
         }
       }
     },
     emphasis: {
-      backgroundColor: cardEmphasisBackgroundColor,
```

Comment:
Why is this being removed. I think there are a lot of unnecessary changes to host config.

Reply:
I believe you are referring to cardEmphasisBackgroundColor(?). As I mention in the PR description, setting actual values (vs null) in the HostConfig compiles to inline CSS on those elements. This then requires the developer to have to append '!important' to any styling changes for them to apply as inline has higher priority.
As these values are set in the defaultStyleOptions file, it's actually redundant and unnecessary to specify them here, as well. Plus (as mentioned), it amounts to extra work for the dev.